api4/plugin.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 const (
-	MaximumPluginFileSize = 50 * 1024 * 1024
+	MaximumPluginFileSize = 75 * 1024 * 1024
```

Comment:
Should we be increasing the size just for the developers or when testing is enabled or for everyone?